### ISSUE TITLE
XCODE 15 update: Parse Issue (Xcode): Could not build module 'WebKit'

### ISSUE THREAD

**Issue by alnaughty** [pichillilorenzo/flutter_inappwebview#1807]
- [x] I have read the [Getting Started](https://inappwebview.dev/docs/intro/) section
- [x] I have already searched for the same problem

## Environment

| Technology      | Version       |
|-----------------| ------------- |
| Flutter version |               |
| Plugin version  |               |
| Android version |               |
| iOS version     |               |
| macOS version   |               |
| Xcode version   |               |

Device information: 

## Description

**Expected behavior: Able to build ios apps with no problem/issue** 

**Current behavior: since updating to xcode 15 it crashes and throws `Parse Issue (Xcode): Could not build module 'WebKit'`**

**Comment by MansoorImtiaz** [comment]
I have the same issue

**Comment by tanmoy27112000** [comment]
add this code to the end of the pod file 

```
post_integrate do |installer|
  compiler_flags_key = 'COMPILER_FLAGS'
  project_path = 'Pods/Pods.xcodeproj'

  project = Xcodeproj::Project.open(project_path)
  project.targets.each do |target|
    target.build_phases.each do |build_phase|
      if build_phase.is_a?(Xcodeproj::Project::Object::PBXSourcesBuildPhase)
        build_phase.files.each do |file|
          if !file.settings.nil? && file.settings.key?(compiler_flags_key)
            compiler_flags = file.settings[compiler_flags_key]
            file.settings[compiler_flags_key] = compiler_flags.gsub(/-DOS_OBJECT_USE_OBJC=0\s*/, '')
          end
        end
      end
    end
  end
  project.save()
end
```

this solved my issue

**Comment by dibyanshu-mohanty** [comment]
> add this code to the end of the pod file
> 
> ```
> post_integrate do |installer|
>   compiler_flags_key = 'COMPILER_FLAGS'
>   project_path = 'Pods/Pods.xcodeproj'
> 
>   project = Xcodeproj::Project.open(project_path)
>   project.targets.each do |target|
>     target.build_phases.each do |build_phase|
>       if build_phase.is_a?(Xcodeproj::Project::Object::PBXSourcesBuildPhase)
>         build_phase.files.each do |file|
>           if !file.settings.nil? && file.settings.key?(compiler_flags_key)
>             compiler_flags = file.settings[compiler_flags_key]
>             file.settings[compiler_flags_key] = compiler_flags.gsub(/-DOS_OBJECT_USE_OBJC=0\s*/, '')
>           end
>         end
>       end
>     end
>   end
>   project.save()
> end
> ```
> 
> this solved my issue

Thanks a lot man. It worked for me as well

**Comment by mrhaubrich** [comment]
> add this code to the end of the pod file
> 
> ```
> post_integrate do |installer|
>   compiler_flags_key = 'COMPILER_FLAGS'
>   project_path = 'Pods/Pods.xcodeproj'
> 
>   project = Xcodeproj::Project.open(project_path)
>   project.targets.each do |target|
>     target.build_phases.each do |build_phase|
>       if build_phase.is_a?(Xcodeproj::Project::Object::PBXSourcesBuildPhase)
>         build_phase.files.each do |file|
>           if !file.settings.nil? && file.settings.key?(compiler_flags_key)
>             compiler_flags = file.settings[compiler_flags_key]
>             file.settings[compiler_flags_key] = compiler_flags.gsub(/-DOS_OBJECT_USE_OBJC=0\s*/, '')
>           end
>         end
>       end
>     end
>   end
>   project.save()
> end
> ```
> 
> this solved my issue

Thanks, also worked for me!

**Comment by egor-slatvinskiy-idf** [comment]
Please share a fix if the project has build configurations or multiple targets

**Comment by pichillilorenzo** [comment]
Released new version `5.8.0` and `6.0.0-beta.25` with the fix!

**Comment by Dicksenz** [comment]
Is there a fix for older version of Flutter SDK ?

**Comment by iuzairaslam** [comment]
This solved the issue:

**Comment by mehsarioglu** [comment]
> add this code to the end of the pod file
> 
> ```
> post_integrate do |installer|
>   compiler_flags_key = 'COMPILER_FLAGS'
>   project_path = 'Pods/Pods.xcodeproj'
> 
>   project = Xcodeproj::Project.open(project_path)
>   project.targets.each do |target|
>     target.build_phases.each do |build_phase|
>       if build_phase.is_a?(Xcodeproj::Project::Object::PBXSourcesBuildPhase)
>         build_phase.files.each do |file|
>           if !file.settings.nil? && file.settings.key?(compiler_flags_key)
>             compiler_flags = file.settings[compiler_flags_key]
>             file.settings[compiler_flags_key] = compiler_flags.gsub(/-DOS_OBJECT_USE_OBJC=0\s*/, '')
>           end
>         end
>       end
>     end
>   end
>   project.save()
> end
> ```
> 
> this solved my issue

Thanks, it works!

**Comment by yimao009** [comment]
> Is there a fix for older version of Flutter SDK ?

Released new version 5.8.0 and 6.0.0-beta.25 with the fix!

**Comment by MuhammetAkgoz** [comment]
> > Is there a fix for older version of Flutter SDK ?
> 
> Released new version 5.8.0 and 6.0.0-beta.25 with the fix!

Thank for helping. I love you man who I don't know

**Comment by MuhammetAkgoz** [comment]
> Released new version `5.8.0` and `6.0.0-beta.25` with the fix!

Thanks for helping. I love you, man who I don't know

**Comment by dchristo978** [comment]
> Is there a fix for older version of Flutter SDK ?

Facing the same issue. using flutter 2.10.5 and xcode 15.x. Any solution?
